docs/en/Getting-Started-on-Cluster.md

```diff
@@ -0,0 +1,334 @@
+your AWS access information to the Alluxio configuration by adding the keys to the
+`conf/alluxio-site.properties` file. The following commands will update the configuration.
+
+```bash
+$ echo "aws.accessKeyId=AWS_ACCESS_KEY_ID" >> conf/alluxio-site.properties
+$ echo "aws.secretKey=AWS_SECRET_ACCESS_KEY" >> conf/alluxio-site.properties
+```
+
+You will have to replace **AWS_ACCESS_KEY_ID** with your AWS access key id, and
+**AWS_SECRET_ACCESS_KEY** with your AWS secret access key. Now, Alluxio is fully configured for the
+rest of this guide.
+
+## Validating Alluxio environment
+
+Before starting Alluxio, you might want to make sure that your system environment is ready for running
+Alluxio services. You can run the following command to validate environment on all nodes:
+
+```bash
+$ ./bin/alluxio validateEnv all
+```
+
+You can also make the command run only specific validation task. For example,
+
+```bash
+$ ./bin/alluxio validateEnv all ulimit
+```
+
+Will only run validation tasks that check your system resource limits, on all the nodes.
+
+You can check out [this page](Developer-Tips.html) for detailed usage information regarding this command.
+
+## Setting up a distributed storage as UFS
+
+The [local Getting-Started guide](Getting-Started.html) shows the steps of how to start Alluxio with default local file system.
+In a distributed cluster, Alluxio requires a distributed storage system as Under File System.
+In this doc, we use AWS S3 as an example here. If you want to use an existing distributed HDFS as UFS, the setup is similar.
+
+Also, in preparation for using S3 with Alluxio, create a bucket (or use an existing bucket). You
+should also note the directory you want to use in that bucket, either by creating a new directory in
+the bucket, or using an existing one. For the purposes of this guide, the S3 bucket name is called
+`S3_BUCKET`, and the directory in that bucket is called `S3_DIRECTORY`.
+
+You need to configure Alluxio to use S3 as its under storage system by modifying
+`conf/alluxio-site.properties`. The first modification is to specify an **existing** S3
+bucket and directory as the under storage system. You specify it by modifying
+`conf/alluxio-site.properties` to include:
+
+```
+alluxio.underfs.address=s3a://S3_BUCKET/S3_DIRECTORY
+```
+
+Note that in the previous `Configuration for AWS` section, the required AWS credential is already setup on
+all the Alluxio nodes. This ensures those nodes have the access to this S3 bucket.
+
+If you want to enable more advanced feature for S3 UFS, please refer to this [documentation](Configuring-Alluxio-with-S3.html).
+
+## Starting Alluxio
+
+Next, we will format Alluxio in preparation for starting Alluxio. The following command will format
+the Alluxio journal and the worker storage directory in preparation for the master and worker to
+start.
+
+Please make sure this command is run on all the nodes.
```

Comment:
If `conf/workers`, and ssh are configured, then you only have to run this on the master.